pkg/render/intrusiondetection/dpi/dpi_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/tigera/operator/pkg/dns"
 	"github.com/tigera/operator/pkg/render"
 	relasticsearch "github.com/tigera/operator/pkg/render/common/elasticsearch"
+	rmeta "github.com/tigera/operator/pkg/render/common/meta"
```

Comment:
This import is different for the original PR as OperatorNamespace is in [different package](https://github.com/tigera/operator/pull/1531/commits/33d3287b1e9976eaacdce9bb79ef97bb81ffe594)